### PR TITLE
Remove Astropy helpers

### PR DESCRIPTION
**Pull request by stscicrawford**
This removes astropy helpers from the repository and updates to the latest astropy package template.   This should simplify the package and have most of the aspects automatically set up.

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+    numpy116: numpy==1.16.*
+    numpy117: numpy==1.17.*
+    numpy118: numpy==1.18.*
+
+    astropy30: astropy==3.0.*
+    astropy40: astropy==4.0.*
+    astropylts: astropy==4.0.*
+
+    devdeps: git+https://github.com/numpy/numpy.git#egg=numpy
+    devdeps: git+https://github.com/astropy/astropy.git#egg=astropy
```
Comment (rendinam) [inline tox.ini:56]: Do the testing environments actually require `astropy`? [2/2]

File: .travis.yml [hunk 2]
```diff
@@ -1,67 +1,161 @@
...
+        # Try other python versions and Numpy versions. Since we can assume that
+        # the Numpy developers have taken care of testing Numpy with different
+        # versions of Python, we can vary Python and Numpy versions at the same
+        # time.
+
+        - os: linux
+          python: 3.7
+          name: Python 3.7 with astropy 3.0 and Numpy 1.17
+          stage: Comprehensive tests
+          env: TOXENV=py37-test-astropy30-numpy117
```
Comment (rendinam) [inline .travis.yml:121]: Do the testing environments actually require `astropy`? [1/2]

File: README.rst [hunk 3]
```diff
@@ -1,28 +1,25 @@
-The DMS environment for HST calibration software
-------------------------------------------------
+Calibration data pipeline for Hubble Space Telescope Observations
+-----------------------------------------------------------------
 
 .. image:: http://img.shields.io/badge/powered%20by-AstroPy-orange.svg?style=flat
     :target: http://www.astropy.org
     :alt: Powered by Astropy Badge
```
Comment (rendinam) [inline README.rst:6]: Does this package really depend upon astropy? [1/2]

File: setup.cfg [hunk 4]
```diff
@@ -1,51 +1,70 @@
...
 
-[ah_bootstrap]
-auto_use = True
+[options]
+zip_safe = False
+packages = find:
+python_requires = >=3.6
+setup_requires = setuptools_scm
+install_requires =
+    astropy
```
Comment (rendinam) [inline setup.cfg:20]: Does this package really depend upon `astropy`? [2/2]

File: tox.ini (hunk 1 above)
Comment (stscicrawford) [reply tox.ini:56]: No these are just the default, they can probably be updated once we have something to test

**Comment by stscicrawford** [comment]
As it is mention a few times, the initial package template does not require astropy, but the caldp software that will be part of it, will be.    I'm okay to leave this in for right now, and then update it as we add the tests and initial software.

**Pull request merged**